vpdq/python/tests/test_vpdq_hash.py

```diff
@@ -36,7 +34,7 @@ def test_compare_hashes(self):
         Two VPDQ features are considered the same if each line of the hashes are within DISTANCE_TOLERANCE.
         For hashes that have a quality lower than QUALITY_TOLERANCE, the test will skip them for comoparing.
         """
-        d = Path.cwd().parent.parent
+        d = Path.cwd().parent.parent.parent
```

Comment:
???

blocking: I think your root problem is that you shouldn't be using cwd() here, you should be using the directory that the file is in. There should be a magic python constant for this (maybe __MODULE__ or something like that?)

Reply:
Change to Path(__file__).parents[3]